composer.json

```diff
@@ -1,14 +1,18 @@
 {
+	"platform": {
```

Comment:
Needs to go inside `config`
https://github.com/nextcloud/spreed/blob/master/composer.json#L12

And then you need to reset composer.lock now and redo, because it now installed with 8.1 dependencies

Reply:
well spotted, thx